coriolis/osmorphing/redhat.py

```diff
@@ -188,8 +187,13 @@ def _run_dracut(self):
         self._run_dracut_base('kernel')
 
     def _run_dracut_base(self, rpm_base_name):
-        package_names = self._exec_cmd_chroot(
-            'rpm -q %s' % rpm_base_name).decode().splitlines()
+        package_names = []
+        try:
+            package_names = self._exec_cmd_chroot(
+                'rpm -q %s' % rpm_base_name).decode().splitlines()
+        except exception.CoriolisException as ex:
+            LOG.trace(ex)
```

Comment:
Considering the tools have a `self._event_manager`, I'd emit a clear warning like:
`Failed to query kernel package names. Unable to rebuild initrds for the new platform.`
NOTE: please double-check that the "kernel package names" part is indeed accurate with respect to every place `_run_dracut_base` is called.